### PR TITLE
Fix for #9527

### PR DESCRIPTION
**Pull request by revaret**
When, Issue title is more than one line the underlying issue status and number was not aligned properly ,has implemented a first level fix in the **issues.scss** and written media query for the same

### CODE REVIEW

**Comment by stanhu** [comment]
Can you add a screenshot for how things look now?

**Comment by revaret** [comment]
Fixed screenshot 
screen

**Comment by revaret** [comment]
Screenshot after Fix

screen 1

File: app/assets/stylesheets/pages/issues.scss [hunk 1]
```diff
@@ -4,14 +4,24 @@
     position: relative;
 
     .issue-title {
-      margin-bottom: 5px;
       font-size: $list-font-size;
+      margin-bottom: 5px;
+      font-size: 15px;
```
Comment (stanhu) [inline app/assets/stylesheets/pages/issues.scss]: Is mandating the font size really necessary?

**Comment by stanhu** [comment]
Looks good to me. @tsigo do you see anything in here that needs to be tweaked?

File: app/assets/stylesheets/pages/issues.scss [hunk 2]
```diff
@@ -4,14 +4,23 @@
...
-      margin-bottom: 5px;
       font-size: $list-font-size;
+      margin-bottom: 5px;
       font-weight: bold;
+      float: left;
+      width: 97.7%;
+    }
+    
+    @media screen and (max-width: 1100px){
+      .issues-list .issue .issue-title{width: 97%}
```
Comment (tsigo) [inline app/assets/stylesheets/pages/issues.scss]: Please format this style like the others, on its own line, and include a space before the opening `{` for both the selector and the media query.

**Comment by tsigo** [comment]
One minor thing. Thank you for the fix!

File: app/assets/stylesheets/pages/issues.scss (hunk 2 above)
Comment (revaret) [inline app/assets/stylesheets/pages/issues.scss]: Done that .

**Comment by stanhu** [comment]
Thanks! Can you squash this into one commit?

**Comment by revaret** [comment]
Have Squashed the commits

**Comment by randx** [comment]
Sorry but this breaks UI for small screens. I revert it

**Comment by randx** [comment]
Here is screenshot of problem introduced by this PR screenshot 2015-08-17 18 29 07

**Pull request merged**